tembo-cli/src/cmd/apply.rs

```diff
@@ -748,31 +754,36 @@ pub fn get_rendered_dockerfile(
     Ok(rendered_dockerfile)
 }
 
-fn get_postgres_config(instance_settings: &InstanceSettings) -> String {
+fn get_postgres_config(
+    instance_ps_config: Option<HashMap<String, Value>>,
+    postgres_configs: &std::option::Option<Vec<ControllerPgConfig>>,
+) -> String {
     let mut postgres_config = String::from("");
     let qoute_new_line = "\'\n";
     let equal_to_qoute = " = \'";
-    let stack_type = ControllerStackType::from_str(instance_settings.stack_type.as_str())
-        .unwrap_or(ControllerStackType::Standard);
-
-    let stack = get_stack(stack_type);
-
-    if stack.postgres_config.is_some() {
-        for config in stack.postgres_config.unwrap().iter() {
-            postgres_config.push_str(config.name.as_str());
-            postgres_config.push_str(equal_to_qoute);
-            postgres_config.push_str(format!("{}", &config.value).as_str());
-            postgres_config.push_str(qoute_new_line);
+    if postgres_configs.is_some() {
+        for p_config in postgres_configs.clone().unwrap().into_iter() {
+            match p_config.value {
+                ControllerConfigValue::Single(val) => {
+                    postgres_config.push_str(p_config.name.as_str());
+                    postgres_config.push_str(equal_to_qoute);
+                    postgres_config.push_str(&val);
+                    postgres_config.push_str(qoute_new_line);
+                }
+                ControllerConfigValue::Multiple(vals) => {
```

Comment:
you may consider a test case in a future PR that checks the extension citus is listed first in shared_preload_libraries after being enabled on top of a Stack that includes other shared_preloads, like vectordb